.storybook/main.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  stories: [
+    "../stories/**/*.stories.mdx",
```

Comment:
Are you importing the button from stories or from components?

Reply:
Thanks, I had already changed that, but somehow my second commit did not go through, updated it now! 🎉